Consumer/ConsumerFactory.php

```diff
@@ -0,0 +1,42 @@
+<?php
+namespace M6Web\Bundle\KafkaBundle\Consumer;
+
+use M6Web\Bundle\KafkaBundle\Consumer\ConsumerManager;
+
+/**
+ * Class ConsumerFactory
+ * @package M6Web\Bundle\KafkaBundle\Consumer
+ *
+ * A class to produce a consumer
+ */
+class ConsumerFactory
+{
+    /**
+     * @param array $brokers
+     * @param array $topics
+     * @param int   $logLevel
+     * @param array $consumerConf
+     *
+     * @return \M6Web\Bundle\KafkaBundle\Consumer\ConsumerManager
+     */
+    public function getConsumer(array $brokers, array $topics, int $logLevel = LOG_WARNING, array $consumerConf = []): ConsumerManager
+    {
+        $conf = new \RdKafka\Conf();
+
+        array_walk($consumerConf, function ($conf, $value) {
+            $conf->set($conf, $value);
+        });
+
+        $kafkaConsumer = new ConsumerManager();
+        $kafkaConsumer
+            ->setConsumer(new \RdKafka\Consumer($conf))
+            ->setLogLevel($logLevel)
+            ->addBrokers(implode(',', $brokers));
+
+        array_walk($topics, function ($topic, $topicName) use ($kafkaConsumer) {
+            $kafkaConsumer->addTopic($topicName, $topic['conf']);
+        });
```

Comment:
```php
array_map(
    [$kafkaConsumer, 'addTopic'],
    array_keys($topics),
    array_column($topics, 'conf')
);
```
:trollface: 
Or why not *just*
```php
foreach($topics as $topicName => $topic) {
    $kafkaConsumer->addTopic($topicName, $topic['conf']);
}
```
(since the callback is never used again?)

Anyway, not blocking of course